src/main/java/com/endava/soa_5/OOP/Feline.java

```diff
@@ -0,0 +1,57 @@
+package com.endava.soa_5.OOP;
+
+public class Feline extends Mammal implements Ability {
+
+    private final double growingSpeed = 1.36;
```

Comment:
Try to avoid initializing properties outside the constructor if they aren't static.

Reply:
Done!